README.md

```diff
@@ -11,11 +11,18 @@
 This section assumes a working python 3.8 environment with Pipenv installed.
 
 * `cp ec_api/settings/local.py.example ec_api/settings/local.py`
-* Install Python dependencies: `pipenv install --dev`
+* Install Python dependencies:
+    * `pipenv install --dev`
+    * `pip install -r api_endpoints/v1_postcode_lookup/requirements.txt`
 * Run the test suite: `pytest`
 * Run lint checks: `ruff .`
 * Auto-format: `ruff format .`
 
+## Run application
+
+- Frontend: `./mange.py runserver`
+- API: `pipenv run start`
+
```

Comment:
In f71864e3 I've done a little bit to make it a bit easier to spin up the application and document some things that I had to work out. I think there are more improvements I would like to make to the way we're handling dependencies and invoking up the application as this is a bit messy. That said, I'm going to stop pulling the thread here and make those things another PR for another day. A sub-optional process which is a bit more clearly documented is at least leaving things a little better than I found them.